HoloPipelines/pipelines/pipelineAbdomenDicom2glb.py

```diff
@@ -0,0 +1,31 @@
+from components import compDicom2nifti
+from components import compHttpRequest
+from components import compNifti2numpy
+from components import compSeparateNumpy
+from components import compNumpyTransformation
+import sys
+import logging
+
+
+def main(
+    inputDicomPath, outputGlbFolderPath, segmentationModelUrl, resolutionLimit=300
+):
+    generatedNiftiPath = compDicom2nifti.main(inputDicomPath, "_temp.nii")
```

Comment:
This file `_temp.nii` is left in the base directory of the project. Do keep all temp files out of the base directory of the project. Also it's best they are cleaned up when unneeded.

Reply:
Changed from base dir to `medicalScans/nifti`. We plan to do clean up in the PIPELINE/API branch as Niel has mentioned in one of the PR comment for API, where we will have all the downloaded and temp files in a folder the job is allocated to i.e. `workSpace/<job id>/nifti/_temp.nii` (name not final). With this we can just delete the whole `<job id>` dir once job is finished (or failed).